wintermotes/index.html

```diff
@@ -20,5 +33,51 @@
 	<!-- 8. All messages must be printed in the order they where submitted -->
 
 	<div id="message-container"></div>
+
+	<script language="Javascript">
+	var messages = []
+	var container = document.getElementById('message-container')
+
+	function showMessage(){
+		var message = document.getElementById('messageValue')
+		
+		
+
+		if(validateString(message.value)){
+			container.innerHTML = container.innerHTML + "<p>" + message.value + "</p>"
+			messages.push(message.value)
+		} else {
+			console.log("String had number")
+		}
+	}
+
+	function validateString(stringToValidate){
+		// You could also use the .js built-in functions, e.g string.NaN or isNumeric 
+		console.log("Validate String called with value: " + stringToValidate);
+		var numbers= ['1', '2', '3', '4', '5', '6', ' 7', '8', '9'];
+		// var special cars[...]
+		for(var i = 0; i<stringToValidate.length; i++){
```

Comment:
Given that the criteria changed to "the string must not contain other than the chars [a-z]", how would you then have solved this?